frontend/src/entities/jobs/hpo/create/training-definitions/training-job-definition.tsx

```diff
@@ -264,6 +258,13 @@ export function TrainingJobDefinition (props: TrainingJobDefinitionProps) {
                                                             ) +
                                                         ` (${index === 0 ? 'min value' : 'max value'})`,
                                                 });
+                                            } else if (value && ((index === 1 && !parameterRange.MinValue) || (index === 0 && !parameterRange.MaxValue))) {
```

Comment:
Wouldn't this add an issue if MinValue/MaxValue is `0` even though that could be valid?

Reply:
<img width="953" alt="Screenshot 2024-06-24 at 4 43 04 PM" src="https://github.com/awslabs/mlspace/assets/24695748/e0e898f5-491b-4687-9245-f2dd6dd4dc9f">

Like this?

Reply:
Assuming `0`  is allowed in the range, if the `MaxRange = 1` (or whatever) is valid wouldn't it still show as invalid because `!parameterRange.MinValue` would evaluate to true?

Reply:
To be clear these are all being seen as String values, so 0 is "0", if you're concerned about the number 0 being an un-truthy value